samples/ImplicitFlow/AureliaApp/scripts/text.js

```diff
@@ -0,0 +1,391 @@
+/**
+ * @license RequireJS text 2.0.14 Copyright (c) 2010-2014, The Dojo Foundation All Rights Reserved.
+ * Available via the MIT or new BSD license.
+ * see: http://github.com/requirejs/text for details
+ */
+/*jslint regexp: true */
+/*global require, XMLHttpRequest, ActiveXObject,
+  define, window, process, Packages,
+  java, location, Components, FileUtils */
+
+define(['module'], function (module) {
```

Comment:
@PinpointTownes That's a good question. 

* aurelia.json defines it as a `build.loader.plugin` that is associated with HTML and CSS files. 
* aurelia.json also imports it as a dependency from scripts instead of as a module. 

I am not sure why the aurelia CLI scaffolds projects that way. I'll look into that.